src/DynamoCoreWpf/ViewModels/Core/StateMachine.cs

```diff
@@ -802,6 +812,16 @@ internal bool HandlePortClicked(PortViewModel portViewModel)
 
                 if (this.currentState != State.Connection) // Not in a connection attempt...
                 {
+                    if (Keyboard.Modifiers == ModifierKeys.Alt)
+                    {
+                        portViewModel.NodeAutoCompleteCommand.Execute(portViewModel);
```

Comment:
@QilongTang I think we should use `if(portViewModel.NodeAutoCompleteCommand.CanExecute(portViewModel))` as a check here. This will make a call to `CanAutoComplete`. That's how the `DelegateCommand` pattern is supposed to work.

Reply:
Make sense! Will update